queries/models/planejamento/CHANGELOG.md

```
@@ -0,0 +1,278 @@
+
+### Alterado
+
+- Altera o tipo_os de `2025-03-08` e `2025-03-08` no modelo `aux_calendario_manual.sql` para `Extraordinária - Verão` (https://github.com/prefeitura-rio/pipelines_rj_smtr/pull/502)
+
+## [1.3.5] - 2025-03-27
+
+### Corrigido
+
+- Corrigido o tipo_dia de `2025-03-02` no modelo `aux_calendario_manual.sql` (https://github.com/prefeitura-rio/pipelines_rj_smtr/pull/501)
+
+## [1.3.4] - 2025-03-25
```

markdownlint-cli2 (0.21.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)